### PR TITLE
Update readmes to match common design (Network Analysis category) 2/2

### PR DESCRIPTION
**Pull request by JonLavi**

### CODE REVIEW

File: network_analysis/offline-routing/README.md [hunk 1]
```diff
@@ -1,36 +1,38 @@
...
 
 ## How to use the sample
 
-Left-click near a road to add a stop to the route. A number graphic will show its order in the route. After adding at least 2 stops, a route will display. Use the combo box in the top left corner to choose between the travel modes "Fastest" and "Shortest" (how the route is optimized). To move a stop, right-click the graphic to select it, then move your mouse to reposition, and finally right-click again to set the new position. The route will update on-the-fly while moving stops. The green box marks the boundary of the route geodatabase.
+Click near a road to start adding a stop to the route, click again to place it on the map. A number graphic will show its order in the route. After adding at least 2 stops, a route will display. Choose "Fastest" or "Shortest" from the drop down menu to control how the route is optimized. Use a secondary click on stops to select them, and then primary-click to re-place them. The route will update on-the-fly while moving stops. The green box marks the boundary of the routable area provided by the offline data.
```
Comment (Rachael-E) [inline network_analysis/offline-routing/README.md]: this section isn't quite right, partially reverting back to original instructions
```suggestion
Left-click near a road to add a stop to the route. A number graphic will show its order in the route. After adding at least 2 stops, a route will display. Choose "Fastest" or "Shortest" from the drop down menu to control how the route is optimized. To move a stop, right-click the graphic to select it, move your mouse to reposition and finally right-click again to set the new position. The route will update on-the-fly while moving stops. The green box marks the boundary of the routable area provided by the offline data.
```

File: network_analysis/routing-around-barriers/README.md [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
 
 ## How to use the sample
 
-Use the Edit Mode toggle buttons to select whether to add Stops or Barriers to the route. Click 'Determine Route' to find the route and display it. Select 'Find best sequence' to allow stops to be re-ordered in order to find an optimum route. Select 'Preserve first stop' to preserve the first stop. Select 'Preserve last stop' to preserve the last stop.
+Use the 'Edit Mode' toggle buttons to select whether to add Stops or Barriers to the route. The route will be solved automatically as you add stops and barriers, and information about the length of the route and directions will be shown in the controls area. Select 'Find best sequence' to allow stops to be re-ordered in order to find an optimum route. Select 'Preserve first stop' to preserve the first stop. Select 'Preserve last stop' to preserve the last stop. You can use the 'Reset' button to reset the sample.
```
Comment (Rachael-E) [inline network_analysis/routing-around-barriers/README.md]:
```suggestion
Use the "Edit Mode" toggle buttons to select whether to add Stops or Barriers to the route. The route will be solved automatically as you add stops and barriers, and information about the length of the route and directions will be shown in the controls area. Select 'Find best sequence' to allow stops to be re-ordered in order to find an optimum route. Select 'Preserve first stop' to preserve the first stop. Select 'Preserve last stop' to preserve the last stop. You can use the "Reset" button to reset the sample.
```

File: network_analysis/service-area-task/README.md [hunk 3]
```diff
@@ -1,43 +1,38 @@
...
-    * Hit the barrier button again to finish drawing barrier.
-    * Hitting any other button will also stop the barrier from drawing.
-4. To show service areas around facilities that were added, click show service areas button.
-5. Reset button, clears all graphics and reset ServiceAreaTask.
+You might calculate the region around a hospital in which ambulances can service in 30 min or less.
 
-## How it works
+## How to use the sample
+
+In order to find any service areas at least one facility needs to be added to the map view. Add facilities to the map by clicking the 'Add Facilities' button and then clicking anywhere on the map. To add a barrier, click the 'Add Barrier' button, and click multiple locations on map. Clicking any other button will stop the barrier from drawing. To show the service areas around facilities that were added, click 'Show Service Areas' button. The 'Reset' button clears all graphics and resets the service area task.
```
Comment (Rachael-E) [inline network_analysis/service-area-task/README.md]: I think we've been learning towards using double quotes for button names

**Review by Rachael-E — COMMENTED** [review]
suggested change here to have a look at, thanks!

File: network_analysis/service-area-task/README.md (hunk 3 above)
Comment (JonLavi) [reply network_analysis/service-area-task/README.md]: Yep, this was inconsistent. Updated!

**Review by Rachael-E — APPROVED** [review]

**Pull request merged**